.toys/.toys.rb

```diff
@@ -135,6 +135,17 @@ def run
   include :exec, e: true
   include :terminal
 
+  Signal.trap("SIGINT") do
+    puts "SIGINT received. Cleaning up processes."
+    toys_gen_pids = `ps aux`.each_line.filter_map { |line| line.split[1].to_i if line.match?(/ruby.*protoc/)}
```

Comment:
`/ruby.*protoc/` seems pretty lenient. Are you confident this won't be likely to hit false positives?

Reply:
See other comment, changed to `protoc-gen-ruby_`. Within the `gen` tool, I think it is safer wrt false positives.